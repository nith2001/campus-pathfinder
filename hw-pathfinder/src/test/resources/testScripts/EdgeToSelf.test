# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

CreateGraph simpleGraph

AddNode simpleGraph A

AddEdge simpleGraph A A 0.0

FindPath simpleGraph A A

ListNodes simpleGraph