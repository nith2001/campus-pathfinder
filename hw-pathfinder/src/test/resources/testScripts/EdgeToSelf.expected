# Find a path with Dijkstra's algorithm when
# the path found by BFS is different.

created graph simpleGraph

added node A to simpleGraph

added edge 0.000 from A to A in simpleGraph

path from A to A:
total cost: 0.000

simpleGraph contains: A